spi_nand_flash/examples/nand_flash_debug_app/main/Kconfig.projbuild

```diff
@@ -0,0 +1,9 @@
+menu "SPI Example Configuration"
+
+    config EXAMPLE_FORMAT_IF_MOUNT_FAILED
```

Comment:
Why do we need this option? This example doesn't (properly) use any file-system to demonstrate the statistics collection

Reply:
Yes, I forgot to remove it. I will update it. Thank you